commit e72159b29f656ccdf0e0643612f80a40ab3ffde4
Author: Fernando Perez <Fernando.Perez@berkeley.edu>
Date:   Fri Jun 28 13:45:12 2013 -0500

    Change doctest that was failing due to python2/3 print syntax issues.

diff --git a/IPython/testing/decorators.py b/IPython/testing/decorators.py
index 5d9edcb..28d368f 100644
--- a/IPython/testing/decorators.py
+++ b/IPython/testing/decorators.py
@@ -128,15 +128,17 @@ def make_label_dec(label,ds=None):
     --------
 
     A simple labeling decorator:
-    >>> slow = make_label_dec('slow')
-    >>> print slow.__doc__
-    Labels a test as 'slow'.
 
+    >>> slow = make_label_dec('slow')
+    >>> slow.__doc__
+    "Labels a test as 'slow'."
+    
     And one that uses multiple labels and a custom docstring:
+    
     >>> rare = make_label_dec(['slow','hard'],
     ... "Mix labels 'slow' and 'hard' for rare tests.")
-    >>> print rare.__doc__
-    Mix labels 'slow' and 'hard' for rare tests.
+    >>> rare.__doc__
+    "Mix labels 'slow' and 'hard' for rare tests."
 
     Now, let's test using this one:
     >>> @rare
